float64 depth
float64 bb_x
float64 bb_y
float64 frame_x
float64 frame_y

